### DISCUSSION TITLE
Initial prototype concept development discussion

### DISCUSSION THREAD

**Discussion by sorensensig** [digdir/insights-hub#8] [Ideas]
### Connection to upcoming research
How will this repository connect to open-source research that has yet to be done? Research that can be listed by one actor as needed, but not initialised, that another actor can take on if they have the means to. How will these two systems work in tandem and where do they both exist in relation to each other?

## Requirements

For this concept to work open-source outside contributors will have to have certain rights in the repository
- [x] Check which restrictions outside collaborators have in Github by default
A collaborator can create new issues, edit their issues, but cannot edit other peoples issues. Moreover, they are not able to connect their issue to a project, labels etc.
- After exploring various Github capabilities, we have come to understand that there is no easy way to give external contributors access to setting labels, projects or similar to their own issue without also giving them full write access to other issues.

We might want to have curated lists for findings, insights and research outcomes outside of searching through issues in the repositories.
- [ ] Explore whether projects work for curated lists

We want to offer users an easy way of adding to the insights hub.
- [ ] Explore how templating can work for users to add findings, insights and research outcomes to the repository

## Templating
We need the following templates:

- [x] Research outcomes
- [x] Findings
- [x] Insights

### Research Outcomes
**Fields:**
- [ ] What (textarea)
- [ ] Who (owner of research)
- [ ] Media format (select list) + label
- [ ] Research methods (select list) + label
- [ ] Sample size (select list) + label
- [ ] Date created (text area)
- [ ] URL/Link to study (text area)
- [ ] Available languages (select list)
- [ ] Keywords (text area)

### Findings
**Fields:**
- [ ] What (text area)
- [ ] Where (text area)
- [ ] Who (text area)
- [ ] Type (select list)
- [ ] Experience vector (select list)
- [ ] Examples (text area)
- [ ] Relevant research (text area with issues linked)

### Insights
**Fields:**
- [ ] What (text area)
- [ ] Findings (text area with issues linked)
- [ ] Recommendations / Opportunities (text area)
- [ ] Type (select list)
- [ ] Relevant research (text area)

## Project
How can we make the project work for both users looking for curated information as well as the maintainer?

- [ ] Tracked by column (list out all issues tracking the issue)

**User**
- [ ] Which fields from the Insights, Findings and Research Outcomes should be visualised in seperate columns in the project views?
- [ ] Which fields should be added to the project that is not part of the issue information?
- [ ] Which languages should we support and how? Bilingual titles (Bokmål and English?)

**Maintainer**
- [ ] What tasks does the maintainer need to do on a regular basis?

## Effect

How do we measure the impact of the insights hub?
- [ ] OECD's good practice principles
- [ ] Sustainability development goals
- [ ] EIF + STEEPLE

**Comment by indiamaydesign** [comment]
Migrating over all the [test entries](https://github.com/orgs/Altinn/projects/84/views/1?pane=issue&itemId=24860285) to the base here beginning with Funn - For mye ligger bak innlogging med BankID / Too much is behind login with BankID

**Reply by indiamaydesign** [reply]
Everything has been done now except for keywords 🎉
